src/config.h

```diff
@@ -104,11 +114,11 @@ typedef struct {
   size_t indexPoolSize;
   int poolSizeNoAuto;  // Don't auto-detect pool size
 
-// #ifdef POWER_TO_THE_WORKERS
+#ifdef MT_BUILD
   size_t numWorkerThreads;
-  int alwaysUseThreads;
+  MTMode mt_mode;
   size_t tieredVecSimIndexBufferLimit;
-// #endif
+#endif
```

Comment:
I think that the `#ifdef` was commented out on purpose, and the build error that you have in coordinator is due to this change...